.github/workflows/ci.yml

```diff
@@ -539,7 +539,7 @@ jobs:
           7z x cmake-$Env:CMAKE_VERSION-win64-x64.zip
           echo "cmake-$Env:CMAKE_VERSION-win64-x64/bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8
       - name: Build
-        run: cmake --version && cppbuild/cppbuild
+        run: cmake --version && cppbuild/cppbuild --parallel
```

Comment:
There is no `--parallel` option anymore and thus the script will fail with an error.